parkkihubi/settings.py

```diff
@@ -159,7 +159,7 @@
     'ALLOWED_VERSIONS': ('v1',),
     'DEFAULT_FILTER_BACKENDS': ('django_filters.rest_framework.DjangoFilterBackend',),
     'EXCEPTION_HANDLER': 'parkings.exception_handler.parkings_exception_handler',
-    'PAGE_SIZE': 20,
+    'PAGE_SIZE': 100,
```

Comment:
This also affects the public API, is that something we want?

Reply:
Yes, it will. IMHO 100 is still much better default. It can be changed with page_size param anyway.